addons/hw_posbox_homepage/static/src/app/components/dialog/HandlerDialog.js

```diff
@@ -0,0 +1,191 @@
+/* global owl */
+
+import useStore from "../../hooks/useStore.js";
+import { BootstrapDialog } from "./BootstrapDialog.js";
+import { LoadingFullScreen } from "../LoadingFullScreen.js";
+
+const { Component, xml, useState } = owl;
+
+export class HandlerDialog extends Component {
+    static props = {};
+    static components = { BootstrapDialog, LoadingFullScreen };
+
+    setup() {
+        this.store = useStore();
+        this.state = useState({
+            initialization: true,
+            waitRestart: false,
+            loading: false,
+            handlerData: {},
+            globalLogger: {},
+        });
+    }
+
+    onClose() {
+        this.state.initialization = [];
+        this.state.handlerData = {};
+    }
+
+    async getHandlerData() {
+        try {
+            const data = await this.store.rpc({
+                url: "/hw_posbox_homepage/log_levels",
+            });
+            this.state.handlerData = data;
+            this.state.globalLogger = {
+                "iot-logging-root": data.root_logger_log_level,
+                "iot-logging-odoo": data.odoo_current_log_level,
```

Comment:
if you compare to a regular iot box, this is indeed the POST value name, but the field name itself is just "root" and "odoo" see:
![image](https://github.com/user-attachments/assets/e3bdf240-e4c6-4c3b-95cd-a0944d3a10ae)
and their level should NOT be set to "Same as Odoo" (as they are the parent)